rcsbsearch/search.py

```diff
@@ -205,7 +207,7 @@ class Terminal(Query):
     operator: Optional[str] = None
     value: Optional[TValue] = None
     service: str = "text"
-    negation: bool = False
+    negation: Optional[bool] = False
```

Comment:
Ideally I think we should swap this to `None` as well, but I was afraid of breaking existing workflows which depend on this default. Instead I am passing `negation=None` in the `TextQuery` class `super().__init__`.